pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -371,6 +373,31 @@ public void healthcheck(@Suspended AsyncResponse asyncResponse) throws Exception
                     asyncResponse.resume("ok");
                 }
             });
+
+    }
+
+    @GET
+    @Path("/healthv2")
```

Comment:
if you have different behavior of endpoint based on topic name then 
v2 endpoint should go under: `org.apache.pulsar.broker.admin.v2.Brokers` and v1 should go under: `org.apache.pulsar.broker.admin.v1.Brokers`

Reply:
done